quadrat/theme.json

```diff
@@ -167,8 +167,8 @@
 			},
 			"post-content": {
 				"padding": {
-					"left": "var(--wp--custom--margin--horizontal)",
-					"right": "var(--wp--custom--margin--horizontal)"
+					"left": "min( var(--wp--custom--margin--horizontal), 5vw)",
```

Comment:
I like the approach to use min here with a px + relative unit comparison. I wonder if the `5vw` should actually be a smaller value, so the padding on mobile is tighter. Right now it seems to only shrink ever so slightly at viewports < 400px.

Reply:
In my testing, `5vw` seems fine to me. This seems good to go. 👍